backend/db/ddl-6-cset_members_items.jinja.sql

```diff
@@ -41,8 +41,8 @@ JOIN {{schema}}concept c ON COALESCE(csm.concept_id, item.concept_id) = c.concep
 WHERE csm.codeset_id IS NOT NULL
    OR item.codeset_id IS NOT NULL;
 
-CREATE INDEX {{optional_index_suffix}}csmi_idx1 ON {{schema}}cset_members_items{{optional_suffix}}(codeset_id);
+CREATE INDEX csmi_idx1{{optional_index_suffix}} ON {{schema}}cset_members_items{{optional_suffix}}(codeset_id);
```

Comment:
## Fix: Suffix position reversed
In `cset_members_items` DDL. Fixed just for internal consistency / linguistic correctness.